app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        Person.people[name] = self
+
+
+def create_person_list(people: dict) -> list:
+    person_list = []
+
+    for person in people:
+        new_person = Person(person["name"], person["age"])
+        person_list.append(new_person)
+
+    for person in people:
+        new_person = Person.people[person["name"]]
+
+        if "wife" in person and person["wife"] is not None:
+            new_person.wife = Person.people[person["wife"]]
+
+        if "husband" in person and person["husband"] is not None:
+            new_person.husband = Person.people[person["husband"]]
```

Comment:
Similar to the `wife` attribute, ensure that the `husband` attribute is defined in the `Person` class to avoid an `AttributeError`.